### ISSUE TITLE
The usage of get_grad_norm_() function

### ISSUE THREAD

**Issue by ZiboZ** [facebookresearch/mae#114]
Thanks a lot for the great work! I have a concern about the purpose of get_grad_norm_() function (utils/misc), it seems be a part of torch.utils.clip_grad_norm() function. However it does not change the gradient/parameter at all but only slow down the training, did I miss something or norm = get_grad_norm_() can be safely removed?
![image](https://user-images.githubusercontent.com/103976483/184507851-079e5974-a723-44dc-863c-9b69f6e911b8.png)
